src/views/compute/DeployVM.vue

```diff
@@ -918,6 +933,8 @@ export default {
         deployVmData.keypair = values.keypair
         deployVmData.name = values.name
         deployVmData.displayname = values.name
+        // step 8: enter setup
+        deployVmData.bootintobios = this.bootintobios
```

Comment:
I think this setting can be at the top step itself where we're getting name, keyboard, etc.

Reply:
ok, kinda makes sense